releasenotes/notes/vf2postlayout-no-better-solution-eb5ced3c8a60ea23.yaml

```
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    VF2PostLayout no distinguishes between 'no solution' and 'no better solution' when determining a
+    Layout for a given quantum circuit. 'no better solution' is set when the initial layout of a quantum
+    circuit is also the optimal one, i.e. incurs the least cost in terms of error rates.
```

Minor: `VF2PostLayout` and `Layout` would best as Sphinx cross-references, so users can click them and jump to the full documentation on them.  The syntax looks like this:
```suggestion
    :class:`.VF2PostLayout` no distinguishes between 'no solution' and 'no better solution' when determining a
    :class:`.Layout` for a given quantum circuit. 'no better solution' is set when the initial layout of a quantum
    circuit is also the optimal one, i.e. incurs the least cost in terms of error rates.
```

---

Sure, I changed it, thanks for the feedback on the release note! :)